rotkehlchen/chain/ethereum/modules/uniswap/v2/common.py

```diff
@@ -89,6 +89,7 @@ def decode_uniswap_v2_like_swap(
         if event.asset == A_ETH and event.event_type == HistoryEventType.RECEIVE:
             received_eth += event.balance.amount
 
+    counterparty_found = False
```

Comment:
Why this variable? Why not just have `counterparty` as `None` and then just populate it if found?